### ISSUE TITLE
Migration to 9.0 can fail when project pre_migration adds is_timesheet column

### ISSUE THREAD

**Issue by NL66278** [OCA/OpenUpgrade#750]
Impacted versions: 9.0

Steps to reproduce: not clear what exact combination of modules installed causes this. At least project

Current behavior: Pre migration of project tries to add is_timesheet column to account_analytic_line. In some cases this column already exists, causing a CRITICAL message and therefore a crash of the migration

Expected behavior: Succesfull migration, wether or not is_timesheet column already exists.

Video/Screenshot link (optional):

**Comment by pedrobaeza** [comment]
The column is created when project_timesheet is not installed, which adds that column: https://github.com/OCA/OpenUpgrade/blob/9.0/addons/project/migrations/9.0.1.1/pre-migration.py#L81.

You should have another module that adds the column by its own. You can put a PR adding a check if the column exists for not creating it again.

**Comment by quangvnm** [comment]
Hello,
We had the same problem while migrating to 9.0. By turning on Postgres's statetment log, we found that at one moment, the followings queries are executed:
```sql
ALTER TABLE "account_analytic_line" ADD COLUMN "is_timesheet" bool
COMMENT ON COLUMN account_analytic_line."is_timesheet" IS 'Is a Timesheet'
```
and at the end (where exception occured):
```sql
ALTER TABLE account_analytic_line ADD is_timesheet boolean
ERROR:  column "is_timesheet" of relation "account_analytic_line" already exists
```

By searching for the comment "Is a Timesheet", we found out that the first statement come from `hr_timesheet` module, in `hr_timesheet\hr_timesheet.py`:
```python
class account_analytic_line(models.Model):
    _inherit = 'account.analytic.line'
    is_timesheet = fields.Boolean(string="Is a Timesheet")
```
We supposed that this column is declared in hr_timesheet instead of project_timesheet?

**Comment by pedrobaeza** [comment]
Fixed in #840
